app/src/main/java/com/cornellappdev/volume/ui/viewmodels/PublicationsViewModel.kt

```diff
@@ -24,7 +24,8 @@ class PublicationsViewModel @Inject constructor(
     ViewModel() {
```

Comment:
Does this PR add the interaction between publications moving between the sections on follow? I do not see the logic for that in the ViewModel

Reply:
yeah this is in the followpublication and unfollowpublication viewModel, but it doesn't load right away